array-methods/for-each.js

```diff
@@ -0,0 +1,6 @@
+const values = [10, 1, 22, 23, 41, 5, 18, 7, 80, 9];
+
+console.log('In order:');
+values.forEach((element) => console.log(element));
+console.log('Reverse order:');
+values.reverse().forEach((element) => console.log(element));
```

Comment:
Log the numbers _without_ using `.reverse()`:

---

![image](https://github.com/annaaksenov/m2-0723-code-solutions/assets/16143771/3dd9e5f0-2705-4cd0-aa72-d319c321bbee)